server/conn.go

```diff
@@ -140,6 +140,9 @@ var (
 	disconnectNormal            = metrics.DisconnectionCounter.WithLabelValues(metrics.LblOK)
 	disconnectByClientWithError = metrics.DisconnectionCounter.WithLabelValues(metrics.LblError)
 	disconnectErrorUndetermined = metrics.DisconnectionCounter.WithLabelValues("undetermined")
+
+	connIdleDurationHistogramStmt = metrics.ConnIdleDurationHistogram.WithLabelValues("stmt")
```

Comment:
so it means `not in transaction`, `stmt` is a little confused.

Reply:
How about change label name to `in-txn`, and the value is `0` or `1`?